Lib/platform.py

```diff
@@ -275,16 +262,13 @@ def _syscmd_ver(system='', release='', version='',
         return system, release, version
 
     # Try some common cmd strings
-    for cmd in ('ver', 'command /c ver', 'cmd /c ver'):
+    for cmd in (['cmd', '/c', 'ver'], ['command', '/c', 'ver'], ):
```

Comment:
Why not use just `['ver']` with `shell=True`?

Reply:
> Why not use just ['ver'] with shell=True?

I wanted to avoid shell=True whenever possible, but ok, let's use shell=True here.